src/lib/util/ReactionHandler.js

```diff
@@ -111,7 +111,8 @@ class ReactionHandler extends ReactionCollector {
 		else return this.stop();
 
 		this.on('collect', (reaction, user) => {
-			reaction.users.remove(user);
+			const clientPermissions = reaction.message.channel.permissionsFor(reaction.message.client);
+			if (clientPermissions.has("MANAGE_MESSAGES")) reaction.users.remove(user);
```

Comment:
To fix the ESLint error:
```suggestion
			if (clientPermissions.has('MANAGE_MESSAGES')) reaction.users.remove(user);
```